packages/mask/src/site-adaptors/mirror.xyz/injection/Tips/PostVerification.tsx

```diff
@@ -0,0 +1,22 @@
+import { MutationObserverWatcher } from '@dimensiondev/holoflows-kit'
+import { Plugin } from '@masknet/plugin-infra'
+import { DefaultWeb3ContextProvider } from '@masknet/web3-hooks-base'
+import { attachReactTreeWithContainer } from '../../../../utils/index.js'
+import { startWatch } from '../../../../utils/startWatch.js'
+import { querySelector } from '../../utils/selectors.js'
+import { TipsButtonWrapper } from './TipsButtonWrapper.js'
+
+function selector() {
+    const authorWallet = location.pathname.split('/')[1].toLowerCase()
+    return querySelector(`#__next a[href$="/address/${authorWallet}" i] div:nth-child(2)`)
```

Comment:
Is authorWallet case sensitive?

Reply:
<img width="720" alt="image" src="https://github.com/DimensionDev/Maskbook/assets/1141198/a88ac74a-d8cd-49fc-af2c-0debeb4a0252">

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors#syntax:~:text=%5Battr%20operator%20value%20i%5D